aiohttp_json_rpc/exceptions.py

```diff
@@ -32,3 +32,31 @@ class RpcInvalidParamsError(RpcError):
 class RpcParseError(RpcError):
     ERROR_CODE = -32700
     MESSAGE = 'Invalid JSON was received'
+
+
+def error_code_to_exception(error_code: int):
+    """
+    Translates a given error code to the corresponding exception.
+
+    Parameters
+    ----------
+    error_code : int
+        A valid RPC error code.
+
+    Returns
+    -------
+        One of the RpcError exception classes defined in this file.
+
+    Raises
+    ------
+    KeyError
+        When the given ``error_code`` does not correspond to any
+        known RPC exception.
+    """
+    # Please see PR #14 for information on why we don't include ``RpcError``
```

Comment:
Hey @tekulvw,

@fscherf asks you to add a better explanation here. (https://github.com/pengutronix/aiohttp-json-rpc/pull/14#issuecomment-338335650)